source/modules/MEI.shared.xml

```diff
@@ -2501,10 +2501,10 @@
     <attList>
       <attDef ident="part" usage="opt">
         <desc xml:lang="en">Indicates the part in which the current feature should appear. Use '%all' when the
-          feature should occur in every part.</desc>
+          feature should occur in every part, or '%none' in none of the parts.</desc>
         <datatype maxOccurs="unbounded">
           <rng:data type="token">
-            <rng:param name="pattern">(%all|#[\i][\c]+)</rng:param>
+            <rng:param name="pattern">(%all|%none|#[\i][\c]*)</rng:param>
           </rng:data>
         </datatype>
         <constraintSpec ident="check_part_attr_all" scheme="schematron">
```

Comment:
Do we need to update the schematron rule as well? It seems like `'%none'` cannot be mixed with other values, too, right?

Reply:
Also: As discussed in #1451, does this value need to be updated for partstaff too?

Reply:
@musicEnfanthen Having this on `partstaff` doesn't seem to make sense. Why would you say something appears in one part but doesn't show up there on any staff?

Reply:
Sure. Just wanted to double-check since it was mentioned in #1451.

What do you think about the schematron rule?